### ISSUE TITLE
Add version number to save file produced

### ISSUE THREAD

**Issue by JLLeitschuh** [WPIRoboticsProjects/GRIP#654]
The version number should be saved in the grip file that we produce.
This will keep us from trying to import a save that we don't support anymore.

**Comment by SamCarlberg** [comment]
Shouldn't we be able to upgrade old saves? Or at least try, and alert the user if it's incompatible.

**Comment by JLLeitschuh** [comment]
Yes. I agree. We just need to be able to tell what version of grip generated the save.
